### PR TITLE
Configmap validation

### PR DESCRIPTION
**Pull request by beezz**
Adding configmap validation see #4

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -11,9 +14,11 @@ import (
 	_ "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/rate_limit/v2"
 	_ "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/redis_proxy/v2"
 	_ "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/tcp_proxy/v2"
+	"github.com/ghodss/yaml"
```
Comment (mattrobenolt) [inline main.go]: Why use a different yaml library here compared to everywhere else?

Comment (beezz) [reply main.go]: Changed to use only one lib everywhere. (was not intentional, vscode tooling pulled this yaml lib instead)

File: configstore.go [hunk 2]
```diff
@@ -32,6 +32,56 @@ type Config struct {
 	services map[string]struct{}
 }
 
+// NewNewConfig initializes config struct
```
Comment (b1naryth1ef) [inline configstore.go]: nit: doc typo here

File: main.go [hunk 3]
```diff
@@ -24,6 +29,27 @@ import (
 const XDS_CONFIGMAP_ENV = "XDS_CONFIGMAP"
 const XDS_LISTEN_ENV = "XDS_LISTEN"
 
+// ReReadFileorStdin returns content of file or stdin.
```
Comment (b1naryth1ef) [inline main.go]: nit: doc typo here

**Review by b1naryth1ef — APPROVED** [review]
LGTM

**Pull request merged**